requirements.txt

```diff
@@ -10,6 +10,8 @@ distro==1.4.0
 Django==3.1.1
 django-filter==2.3.0
 djangorestframework==3.11.1
+djangorestframework-jwt==1.11.0
```

Comment:
which one of these 3 libraries, we are using for authentication?

Reply:
rest_framework_jwt